src/sp_repo_review/checks/pyproject.py

```diff
@@ -241,5 +241,65 @@ def check(pyproject: dict[str, Any]) -> bool:
         return "filterwarnings" in options
 
 
+class PP310(PyProject):
+    "Tests target is test not test (spec13)"
+
+    requires = {"PP301"}
+    url = mk_url("pytest")
+
+    @staticmethod
+    def check(pyproject: dict[str, Any]) -> bool | None:
+        """
+
+        Tests target should be `tests` not `test`
+
+        ```toml
+        [project.optional-dependencies]
+        tests = [
+            'pytest',
+            ...
+        ]
+        ```
+        """
+        if "tool" not in pyproject:
+            return None
+        if "project.optional-dependencies" not in pyproject["tool"]:
+            return None
+        optional_deps = pyproject["tool"]["project.optional-dependencies"]
+        if "tests" in optional_deps:
+            return True
+        return "test" not in optional_deps
```

Comment:
```suggestion
        match pyproject:
            case {"tool": "project" { "optional-dependencies": "tests": _}}:
                return True
            case {"tool": "project" { "optional-dependencies": "test": _}}:
                return False
            case _:
                return True
```

Try pattern matching here. ;)

Reply:
Yeah, I need to practice more my pattern matching :-)

Reply:
SPEC 0 should make that easier now. :)